Draft-OpenChain-Telco-SBOM-Guide_EN.md

```diff
@@ -58,10 +58,11 @@ Package URL (PURL) is a _de facto_ standard to uniquely identify software packag
 ## 3. Requirements
 
 ### 3.1 Data Format
-A Telco SBOM SHALL adhere to the version 2.2 of the SPDX Data Format as standardized in ISO/IEC 5962:2021 and as further described below with respect to the included elements.
+A Telco SBOM SHALL adhere to the version 2.2 of the SPDX Data Format as standardized in ISO/IEC 5962:2021, or to the version 2.3 of the standard, and as further described below with respect to the included elements.
 
 #### 3.1.1 Verification and reference material
-ISO/IEC 5962:2021 Information technology — SPDX® Specification V2.2.1
+* ISO/IEC 5962:2021 Information technology — SPDX® Specification V2.2.1
```

Comment:
2.3 has a hyperlink, should a hyperlink added here too?

Reply:
The hyperlinks are grouped at the end of the document, in the "References" section.